### PR TITLE
Add web caching

### PR DESCRIPTION
**Pull request by zvaraondrej**
**https://github.com/zvaraondrej/awesome-cache**

**The list provides a set of resources focused on the core concepts of caching, related tutorials and web tools. Both browser and server caching are explained. Correct caching mechanism is an important part of a well-built web app. In the same time, understanding caching mechanism is often a confusing task. This list tries to provide high-quality resources for understanding the topic.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by vitalkanev** [comment]
> The repo should have awesome-list & awesome as GitHub topics. I encourage you to add more relevant topics.

<hr/>

> Has an appropriate license.

No `LICENSE` file!

**Comment by zvaraondrej** [comment]
License file added.

**Comment by janisozaur** [comment]
I'd like to point out that the link and title of that list is very misleading. When I saw it, I was initially interested, hoping to find useful information about things like c2c false sharing, data prefetching, vtune or optimising hash maps… only after reading into the list I realised the disappointment, as it is about no such thing.

As an `awesome` user/reader, I'd rather not have this merged until this is addressed.

**Comment by zvaraondrej** [comment]
@janisozaur so do you have any proposition regarding the correct list name?

**Comment by janisozaur** [comment]
Http cache? Web cache? And mentioning that in the title, not few paragraphs into the list and only hidden in some sentence there.

**Comment by janisozaur** [comment]
screenshot_20171013-203615_01

There is nothing in here about web or http or browsers…

**Comment by janisozaur** [comment]
:+1:

**Comment by zvaraondrej** [comment]
Dear janisozaur, thank you for you friendly notes. I have updated the related parts of the repo as well as the PR.

File: readme.md [hunk 1]
```diff
@@ -569,6 +569,7 @@
 - [Crypto Currency Tools & Algorithms](https://github.com/kennethreitz/awesome-coins) — Digital currency where encryption is used to regulate the generation of units and verify transfers.
 - [Open Source Supporters](https://github.com/zachflower/awesome-open-source-supporters) - Companies that offer their tools and services for free to open source projects.
 - [Design Principles](https://github.com/robinstickel/awesome-design-principles) - Create better and more consistent designs and experiences.
+- [Web Caching](https://github.com/zvaraondrej/awesome-web-cache) - Storing of a web resources.
```
Comment (sindresorhus) [inline readme.md]: The title here is not consistent with the list name, and the description is not good enough. Caching is not just about storing a resource.

**Comment by sindresorhus** [comment]
I would like to see more than just a few articles and tools. Include links to community resources (Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community), videos of talks, book, learning material, etc. The list right now is pretty thin on content.

**Comment by sindresorhus** [comment]
> The repo should have awesome-list & awesome as GitHub topics. I encourage you to add more relevant topics.

Not done

> Drop all the A / An prefixes in the descriptions.

Not done

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**